app/config/question-bank.js

```diff
@@ -993,10 +990,11 @@ const questionBank = {
               type: 'number',
               classes: 'govuk-input--width-10',
               prefix: {
-                text: '£'
+                text: '£',
+                error: 'Enter the business turnover'
```

Comment:
@zavies97 can you explain the error here arent we handling this in validation array ??

Reply:
This was supposed to just be in the validation array, but I had added it into two places. Have removed the error from prefix